index.js

```diff
@@ -1 +1,17 @@
-// code your solution here
+const record = [
+  { year: "2015", result: "M"},
+  { year: "2014", result: "N/A"},
+  { year: "2013", result: "L"},
+  //...
+]
+
+function otherFunction(element) {
+  return element.result === 'W'
+}
```

Comment:
What would be a better name for this function besides `otherFunction`? And what's a better name for the argument other than `element`?